src/Command/DetectCommand.php

```diff
@@ -93,7 +93,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         foreach ($paths as $path) {
-            $finder->in($path);
+            $finder->in($path)->followLinks();
```

Comment:
It's a global flag anyway, so it shouldn't make a difference where it is called:
https://github.com/symfony/finder/blob/master/Finder.php#L504